packages/ckeditor5-build-classic/webpack.config.cjs

```diff
@@ -21,7 +21,7 @@ module.exports = {
 
 	output: {
 		// The name under which the editor will be exported.
-		library: 'ClassicEditor',
+		library: 'CKEDITOR',
```

Comment:
The package name is for the classic editor. Maybe ClassicCKEditor?

Reply:
This is how it was done in previous version (of our super build) and owr apps uses CKEDITOR name. I did it to maintain consistency. I agree that naming is not the most suitable. My suggestion is CKeditorInsperaBuild or InsperaCKEditor